Software/grab/GrabberBase.cpp

```diff
@@ -135,8 +135,9 @@ void GrabberBase::grab()
 		}
 	}
 	_lastGrabResult = grabScreens();
-	++grabScreensCount;
+
 	if (_lastGrabResult == GrabResultOk) {
+		++grabScreensCount;
```

Comment:
beware this impacts only the log, not the shown framerate. That is based on handleGrabbedColors

Reply:
Indeed, and now both values should be in phase:
https://github.com/psieg/Lightpack/blob/b4fd772ea3ffd99612103b0f4fec35e94855e390/Software/src/GrabManager.cpp#L666-L668

Reply:
just making sure